site/docs/foundations/iconography/country-symbols.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Country symbols
+layout: DetailTechnical
+---
+
+Country symbols represent a specific country and are used in contexts where geographical identification is necessary. They can be considered icons that are visually representative of a country or inter-governmental organisation. All flags are available in both circle and rectangle shapes.
+
+<Image
+  src="/img/foundations/iconography-countrysymbols-variant.png"
+  alt="Example of circular flag next to rectangular flag"
```

Comment:
Are they symbols or flags

Reply:
Good point thanks, changed it!

Reply:
@amritadesmet There are other references to flag

Reply:
Changed the alt text.